backend/app/utils/__init__.py

```
@@ -0,0 +1,21 @@
+    is_artifact_result,
+)
+from .diff_utils import apply_diff, create_diff, get_version_content
+
+__all__ = [
+    "create_diff",
+    "apply_diff",
+    "get_version_content",
+    "format_artifact_for_history",
+    "extract_artifact_from_result",
+    "is_artifact_result",
+]
```

Ruff (0.14.11)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)